src/index.html

```diff
@@ -20,8 +20,434 @@
       href="styles/index.scss"
     />
   </head>
-
   <body>
-    <h1>Catalog</h1>
+    <div class="catalog">
+      <header class="header">
+        <div>
+          <a
+            href="#"
+            class="img"
+          >
+            <img
+              src="images/logo.png"
+              alt="logo"
+            />
+          </a>
+        </div>
+        <nav class="nav">
+          <a
+            class="nav__link is-active"
+            href="#"
+          >
+            Apple
+          </a>
+          <a
+            class="nav__link"
+            href="#"
+          >
+            Samsung
+          </a>
+          <a
+            class="nav__link"
+            href="#"
+          >
+            Smartphones
+          </a>
+          <a
+            class="nav__link nav__link--laptops"
+            data-qa="nav-hover"
+            href="#"
+          >
+            Laptops & Computers
+          </a>
+          <a
+            class="nav__link"
+            href="#"
+          >
+            Gadgets
+          </a>
+          <a
+            class="nav__link"
+            href="#"
+          >
+            Tablets
+          </a>
+          <a
+            class="nav__link"
+            href="#"
+          >
+            Photo
+          </a>
+          <a
+            class="nav__link"
+            href="#"
+          >
+            Video
+          </a>
+        </nav>
+      </header>
+
+      <main class="main">
+        <div class="card">
+          <div class="cardTop">
+            <div class="cardTop__Img"></div>
+
+            <h1 class="cardTop__h1">
+              APPLE A1419 iMac 27" Retina
+              <br />
+              5K Monoblock (MNED2UA/A)
+            </h1>
+
+            <h2 class="cardTop__h2">Product code: 195434</h2>
+          </div>
+
+          <div class="cardBottom">
+            <div class="starsReviews">
+              <div class="stars stars--5">
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+                <div class="stars__star"></div>
+              </div>
+
+              <div class="reviews">Reviews: 5</div>
+            </div>
+
+            <div class="cardPrice-mix">
+              <div class="cardPrice">
+                <div class="cardPrice__price-1">Price:</div>
+
+                <div class="cardPrice__price-2">$2,199</div>
+              </div>
+            </div>
+
+            <div class="button">
+              <a
+                class="button__element"
+                href="#"
```

Comment:
The `href` attribute for the 'BUY' button is set to `#`, which means it won't lead to any specific page. Ensure this link is correctly set to its intended destination.